### PR TITLE
Pr90x l1t prescales from payloads

### PR DESCRIPTION
**Pull request by rekovic**
90x PR: Updated L1TGlobalUtil to now also access L1T prescales from payloads, using GlobalTag.

Example:
<pre>
cmsRun L1Trigger/L1TGlobal/test/runGTSummary.py 
</pre>

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by Martin-Grunewald** [comment]
@rekovic @apana 
Can you please move the ```cout``` back to ```MessageLogger```?

**Comment by Martin-Grunewald** [comment]
@rekovic @apana
Can you please move the ```cout``` back to ```MessageLogger```?

**Comment by apana** [comment]
Hi Martin,

Sorry I forgot to followup on this request.  I will take care of the cout.

-Len

**Comment by rekovic** [comment]
@Martin-Grunewald, @apana 
> Can you please move the `cout` back to `MessageLogger`?

Done.

File: L1Trigger/L1TGlobal/plugins/L1TGlobalSummary.cc [hunk 1]
```diff
@@ -64,31 +65,42 @@ L1TGlobalSummary::L1TGlobalSummary(const edm::ParameterSet& iConfig){
   dumpRecord_       = iConfig.getParameter<bool>("DumpRecord");
   dumpTriggerResults_ = iConfig.getParameter<bool>("DumpTrigResults");
   dumpTriggerSummary_ = iConfig.getParameter<bool>("DumpTrigSummary");
+  readPrescalesFromFile_ = iConfig.getParameter<bool>("ReadPrescalesFromFile");
   minBx_              = iConfig.getParameter<int>("MinBx");
-  maxBx_              = iConfig.getParameter<int>("MaxBx");     
+  maxBx_              = iConfig.getParameter<int>("MaxBx");
   gtUtil_             = new L1TGlobalUtil(iConfig, consumesCollector(), *this, algInputTag_, extInputTag_);
```
Comment (dmitrijus) [inline L1Trigger/L1TGlobal/plugins/L1TGlobalSummary.cc]: can this become a member? Or at least std::unique_ptr?

**Review by dmitrijus — CHANGES_REQUESTED** [review]

**Comment by dmitrijus** [comment]
+1

I did request a small change, but it was there before this PR as well - the PR does not make it worse.

**Comment by fwyzard** [comment]
sorry @rekovic @davidlange6 , what happened to this PR ?
has i been integrated somewhere else ?

**Comment by davidlange6** [comment]
not that i'm aware of (don't recall seeing a version that worked)

**Comment by rekovic** [comment]
@fwyzard @davidlange6 Correct - the new version is being worked on.

**Pull request closed without merging**